src/BlazorDB/wwwroot/blazorDB.js

```diff
@@ -205,5 +285,136 @@ window.blazorDB = {
+                                delete row[schemaUpdate.columnsToPerformActionOn[i]];
+                            }
+                        });
+                    });
+                    break;
+                case 'subtract':
+                    db.version(dbStore.version).stores(stores).upgrade(trans => {
+                        // Get the table, cast to collection, call necessary function
+                        return trans.table(schemaUpdate.name).toCollection().modify(row => {
+                            // modify each row
+                            for (var i = 0; i < schemaUpdate.upgradeActionParameterList.length; i++) {
+                                row[schemaUpdate.columnsToReceiveDataFromAction[i]] = Math.round((row[schemaUpdate.columnsToPerformActionOn[i]] - schemaUpdate.upgradeActionParameterList[i]) * 100) / 100;
+                                //delete row[schemaUpdate.columnsToPerformActionOn[i]];
+                            }
+                        });
+                    });
+                    break;
+                case 'subtract-delete':
+                    db.version(dbStore.version).stores(stores).upgrade(trans => {
+                        // Get the table, cast to collection, call necessary function
+                        return trans.table(schemaUpdate.name).toCollection().modify(row => {
+                            // modify each row
+                            for (var i = 0; i < schemaUpdate.upgradeActionParameterList.length; i++) {
+                                row[schemaUpdate.columnsToReceiveDataFromAction[i]] = Math.round((row[schemaUpdate.columnsToPerformActionOn[i]] - schemaUpdate.upgradeActionParameterList[i]) * 100) / 100;
+                                delete row[schemaUpdate.columnsToPerformActionOn[i]];
+                            }
+                        });
+                    });
+                    break;
+                case 'delete-column':
+                    db.version(dbStore.version).stores(stores).upgrade(trans => {
+                        // Get the table, cast to collection, call necessary function
+                        return trans.table(schemaUpdate.name).toCollection().modify(row => {
+                            // modify each row
+                            for (var i = 0; i < schemaUpdate.columnsToPerformActionOn.length; i++) {
+                                //row[schemaUpdate.columnsToReceiveDataFromAction[i]] = Math.round((row[schemaUpdate.columnsToPerformActionOn[i]] - schemaUpdate.upgradeActionParameterList[i]) * 100) / 100;
+                                delete row[schemaUpdate.columnsToPerformActionOn[i]];
+                            }
+                        });
+                    });
+                    break;
+                default:
+                    console.warn('default');
```

Comment:
I'd error with something like `Upgrade not implemented` and callback to C# if possible. The C# callback is only if it's easy here, not required.

Reply:
Warning implemented. C# callback not implemented. I will research this and try to implement it on the next iteration.